### DISCUSSION TITLE
Question about creating a group with Injuries

### DISCUSSION THREAD

**Discussion by xXPinkFloyd92Xx** [acemod/ACE3#8448] [Q&A]
Hello,

I want to create a group using a function.sqf:

 

fnc_createGrouplost =
﻿ {
    private ["_pos", "_grp"];
    if (isNil {_this}) exitWith {grpNull;};
    if (typename _this != "ARRAY") exitWith {grpNull;};

    _pos = _this;
    _grp = createGroup west;
    "C_man_hunter_1_F" createUnit [_pos, _grp,"",0.5, "CORPORAL"];
    "C_man_hunter_1_F" createUnit [_pos, _grp,"",0.5, "PRIVATE"];
    "C_man_hunter_1_F" createUnit [_pos, _grp,"",0.5, "PRIVATE"];
    "C_man_hunter_1_F" createUnit [_pos, _grp,"",0.5, "PRIVATE"];

    _grp;
};

-------------------------------------------------------------------------------------

 

This worked fine.

 

I am using ACE medical for this mission (CSAR)

I wanted to add this group with some injured.

 

How can I give individuals a specific inti?

The init I want to add to the unit:

 

this setVariable ["ace_medical_ai_lastFired", 9999999];[this, 0.8, "rightleg", "bullet"] call ace_medical_fnc_addDamageToUnit; [this, 0.8, "leftleg", "bullet"] call ace_medical_fnc_addDamageToUnit

 

any ideas?
Edit 9/21/21
I was given this by someone but it still gives me no injured.
it fixed the broken mission and i was able to see the group but still no Ace injured.
fnc_createGroupLost= {
	if isNil '_this' exitWith {false};
	private _grp = createGroup [WEST, true];
	for "_i" from 0 to 3 do {_grp createUnit ["C_man_hunter_1_F", _this, [], 0, "FORM"]};
	_grp
};

(units ((player modelToWorld [0,5,0]) call fnc_createGrouplost)) apply {
	_x setVariable ["ace_medical_ai_lastFired",9999999];
	[_x, 0.8, "rightleg","bullet"] call ace_medical_fnc_addDamageToUnit;
	[_x, 0.8, "leftleg","bullet"] call ace_medical_fnc_addDamageToUnit
};

Any help will be greatly appreciated, 
Thank you!

**Comment by severgun** [comment]
```sqf
fnc_createGrouplost = {
    params ["_pos", [], [[]]];

    if (_pos isEqualTo []) exitWith {grpNull};

    private _grp = createGroup [west, true];
    "C_man_hunter_1_F" createUnit [_pos, _grp,"",0.5, "CORPORAL"];

    for "_i" from 0 to 3 do {
        "C_man_hunter_1_F" createUnit [_pos, _grp,"",0.5, "PRIVATE"];
    };		

    _grp
};

private _myAwesomeInjuredGroup = [getPos player] call fnc_createGrouplost;

{
    _x setVariable ["ace_medical_ai_lastFired", 9999999, true];
    [_x, 0.8, "rightleg", "bullet"] call ace_medical_fnc_addDamageToUnit;
    [_x, 0.8, "leftleg", "bullet"] call ace_medical_fnc_addDamageToUnit;
} forEach units _myAwesomeInjuredGroup;
```
